### PR TITLE
feat(screenshots): Add screenshots member (closes #477)

### LINKED ISSUE #477: Add screenshots object
Many catalogs / stores of apps include one or more screenshots of an app.

While image dimensions and aspect ratios may vary, it would be useful for the spec to define a place to look for screenshots (similar to the icons object)

This could be an optional member of the manifest, but including a standard place to look for screenshots would be useful in building catalogs of web apps such as https://pwa.rocks/

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@patrickkettner, @RobDolinMS, wdyt?

**Comment by patrickkettner** [comment]
🎉 🎉 🎉 

looks great, two thoughts before the big green button gets pushed

- should there be an optional `platform` or `store` property. Like if Apple only wanted to show iOS style screenshots, or have xbox specific screenshots for the windows store, etc.

- do we want to lock in screenshot as the noun here, in case we want to support video or other content in the future? I would assume the answer is yes, I just want to make sure before we etch it into stone [:

**Comment by marcoscaceres** [comment]
> should there be an optional platform or store property. Like if Apple only wanted to show iOS style screenshots, or have xbox specific screenshots for the windows store, etc.

See #405 ... I think I'll just add `platform` and reuse the existing semantics of platform in the spec till we sort something out.  

> do we want to lock in screenshot as the noun here, in case we want to support video or other content in the future? I would assume the answer is yes, I just want to make sure before we etch it into stone [:

I think using screenshot for both video and static images is ok. People know what they mean and can be differentiated by `type`.

**Comment by patrickkettner** [comment]
LGTM

**Comment by marcoscaceres** [comment]
@patrickkettner thanks for the review! 

@kenchris, if you think it's ok - please merge.

File: index.html [hunk 1]
```diff
@@ -156,7 +156,16 @@
...
   "orientation": "landscape",
   "theme_color": "aliceblue",
-  "background_color": "red"
+  "background_color": "red",
+  "screenshots": [{
+    "src": "screenshots/in-game-1x.jpg",
+    "sizes": "640x480",
+    "type": "image/jpeg"
+  },{
+    "src": "screenshots/in-game-2x.jpg",
```
Comment (kenchris) [inline index.html]: Don't you want one of the examples to use "platform"?

Comment (marcoscaceres) [inline index.html]: yeah, that's probably a good idea... but I'd have to use a fake one.

**Pull request merged**